src/jit/codegenarm.cpp

```diff
@@ -1569,35 +1569,21 @@ void CodeGen::genIntToFloatCast(GenTreePtr treeNode)
     // we expect the front-end or lowering phase to have generated two levels of cast.
     //
     emitAttr srcSize = EA_ATTR(genTypeSize(srcType));
-    noway_assert((srcSize == EA_4BYTE) || (srcSize == EA_8BYTE));
+    noway_assert(srcSize == EA_4BYTE);
 
```

Comment:
Can you please update the comments (here, and in the function header):
```
// genIntToFloatCast: Generate code to cast an int/long to float/double
```
=>
```
// genIntToFloatCast: Generate code to cast an int to float/double
```
and
```
    // We should never see a srcType whose size is neither EA_4BYTE or EA_8BYTE
```
=>
```
    // We only expect a srcType whose size is EA_4BYTE.
```